### PR TITLE
Preserve base64 url suffixes

### PR DESCRIPTION
**Pull request by vincaslt**
Using fragments on base64 urls would previously strip them, leaving SVG icon invalid. This PR makes it so they are preserved in the minimized base64 svg.

Fixes: #807

### CODE REVIEW

File: packages/postcss-svgo/src/index.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { encode, decode } from './lib/url';
 const PLUGIN = 'postcss-svgo';
 const dataURI = /data:image\/svg\+xml(;((charset=)?utf-8|base64))?,/i;
 const dataURIBase64 = /data:image\/svg\+xml;base64,/i;
+const base64String = /(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=|[A-Za-z0-9+/]{4})/;
```
Comment (evilebottnawi) [inline packages/postcss-svgo/src/index.js]: No need regexp, please use URL parser

Comment (vincaslt) [reply packages/postcss-svgo/src/index.js]: OK, did the changes, hopefully I understood what you meant correctly.

**Comment by vincaslt** [comment]
Any new on this? @evilebottnawi

**Comment by evilebottnawi** [comment]
Thanks for the PR

**Pull request merged**